samples/EventImporter/Consumer.Tests/EventImporterTests.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+
+using Consumer.Models;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+using PactNet;
+using PactNet.Matchers;
+using PactNet.Native;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Consumer.Tests
+{
+    public class EventImporterTests
+    {
+        private readonly IMessagePactBuilderV3 _messagePact;
+
+        private readonly PactConfig _config = new PactConfig
+        {
+            LogDir = "../../../logs/",
+            PactDir = "../../../pacts/",
+            DefaultJsonSettings = new JsonSerializerSettings
+            {
+                ContractResolver = new CamelCasePropertyNamesContractResolver()
+            }
+        };
+
+        public EventImporterTests(ITestOutputHelper output)
+        {
+            _config.Outputters = new[]
+            {
+                new XUnitOutput(output)
+            };
+
+            IMessagePactV3 v3 = MessagePact.V3("Event API Consumer V3 Message", "Event API V3 Message", _config);
+            _messagePact = v3.UsingNativeBackend();
+        }
+
+        [Fact]
+        public void GetAllEvents_FromQueue_Should_CreatePact_WithMessages()
+        {
+            var expected = new List<dynamic>
```

Comment:
Would we genuinely expect people to define the event body outside of the interaction definition? Seems a bit odd to do it that way - at first I didn't even realise what this was being used for.